lte/gateway/python/magma/enodebd/state_machines/enb_acs.py

```diff
@@ -40,12 +41,15 @@ class EnodebAcsStateMachine(ABC):
     This ABC is more of an interface definition.
     """
 
-    def __init__(self) -> None:
+    def __init__(self, use_param_key=False) -> None:
```

Comment:
```suggestion
    def __init__(self, use_param_key=False: bool) -> None:
```
minor nit to add typing